pages/pipelines/permissions.md

```diff
@@ -31,6 +31,21 @@ Users who are team maintainers can:
 
 All users in a team have the same level of access to the pipelines in their team. If you need to have more fine grained control over the pipelines in a team, you can create more teams with different permissions.
 
+### Pipeline-level permissions
+
+There are 3 levels of permissions that a team can be given on a pipeline, these are:
```

Comment:
It's a good practice to avoid being specific about the number of items you're about to list to make maintenance easier as items are added or removed later. Perhaps:

```suggestion
You can grant teams the following permissions on a pipeline:
```